darwin/utils.py

```diff
@@ -574,3 +579,27 @@ def is_unix_like_os() -> bool:
         True for Unix-based systems, False otherwise.
     """
     return platform.system() != "Windows"
+
+
+def current_version() -> str:
+    """
+    Returns darwin-py's current version.
+
+    Returns
+    -------
+    str
+        A string in the format of Major.Minor.Bugfix indicating the SDK's current version.
+
+    Raises
+    ------
+    ValueError
+        If the version cannot be retrieved.
+    """
+    with open(Path(__file__).parent.parent / "darwin" / "__init__.py", "r") as f:
```

Comment:
We should import the version number, rather than opening the text file and parse it

Reply:
As far as I know , we can't do that. That is why I open the file and parse it.